packages/browser/src/sdk.ts

```diff
@@ -97,6 +102,10 @@ export function init(options: BrowserOptions = {}): void {
     if (WINDOW.SENTRY_RELEASE && WINDOW.SENTRY_RELEASE.id) {
```

Comment:
m: Should we deprecate and remove in v8?

Reply:
That would kill old webpack plugin compatibility. We definitely could. But I would probably delay it by a major or two.

Reply:
it would be a great way to get people to migrate to newer webpack version, and also we can then point people to this as a feature in the first place.